src/components/Send.tsx

```diff
@@ -1,4 +1,121 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter, RouteComponentProps } from 'react-router';
+import { Form, FormGroup, Input, Label, Button } from 'reactstrap';
+import { getWallets } from '../store/wallet';
+import { AppState } from '../store';
+import api, { CreatePSBTOutputs } from '../api';
+import { Wallet } from '../types';
 
-// TODO: Send page
-export default (() => <>I'm the Send page</>) as React.SFC;
+interface DispatchProps {
+  getWallets: typeof getWallets;
+}
+
+interface ConnectProps {
+  // FIXME: typescript makes me accept possibility of nullness
+  // How can we just demand that this not be null for the component to even load?
+  activeWallet: Wallet | null;
```

Comment:
This is our global wallet state:

```
export interface WalletState {
  wallets: Loadable<Wallet[]>;
  activeWallet: Wallet | null;
}
```

`activeWallet` is either a wallet or null. That makes sense. But how can I have my `Send` component's state not allow `null` value for `activeWallet`? This component doesn't make any sense unless there's an active wallet.

If I remove the `| null` typescript complains.

Reply:
There are a few good ways to do this:
1) Implement a redux selector that just throws if it's not there. That'll fix the typing, and make it really obvious if the component mounts without the necessary prop.
2) Leave it as `| null` and just add `if (!activeWallet) return null;` to the render method.